mailing_list/tests.py

```diff
@@ -0,0 +1,242 @@
+                'david_bonner@harvard.edu',
+                'sapna_mysore@harvard.edu'
+            },
+            add_members={'david_bonner@harvard.edu', 'sapna_mysore@harvard.edu'},
+            delete_members=set(),
+            members_count=4
+        )
+
+    def test_sync_listserv_membership_delete_members_from_listserv(self):
+        self._call_sync_listserv_membership(
+            listserv_email_set={
+                'hiukei_chow@harvard.edu',
+                'david_downs@harvard.edu',
+                'david_bonner@harvard.edu',
+                'sapna_mysore@harvard.edu'
+            },
+            enrolled_email_set={
+                'hiukei_chow@harvard.edu',
+                'david_downs@harvard.edu'
+            },
+            add_members=set(),
+            delete_members={'david_bonner@harvard.edu', 'sapna_mysore@harvard.edu'},
+            members_count=2
+        )
+
+    def test_sync_listserv_membership_add_and_delete_members_from_listserv(self):
+        self._call_sync_listserv_membership(
+            listserv_email_set={
+                'hiukei_chow@harvard.edu',
+                'david_downs@harvard.edu',
+                'david_bonner@harvard.edu'
+            },
+            enrolled_email_set={
+                'hiukei_chow@harvard.edu',
+                'david_downs@harvard.edu',
+                'sapna_mysore@harvard.edu'
+            },
+            add_members={'sapna_mysore@harvard.edu'},
+            delete_members={'david_bonner@harvard.edu'},
+            members_count=3
+        )
+
+    @override_settings(IGNORE_WHITELIST=True)
+    def test_sync_listserv_membership_unsubscribed(self):
+        self._call_sync_listserv_membership(
+            listserv_email_set={
+                'hiukei_chow@harvard.edu',
+                'david_downs@harvard.edu'
+            },
+            enrolled_email_set={
+                'hiukei_chow@harvard.edu',
+                'david_downs@harvard.edu',
+                'david_bonner@harvard.edu',
+                'douglas_hall@harvard.edu'
+            },
+            add_members={'david_bonner@harvard.edu'},
+            delete_members=set(),
+            members_count=3
+        )
+
+    def test_sync_listserv_membership_whitelist_not_set(self):
+        self._call_sync_listserv_membership(
+            listserv_email_set={
+                'hiukei_chow@harvard.edu',
+                'david_downs@harvard.edu'
+            },
+            enrolled_email_set={
+                'hiukei_chow@harvard.edu',
+                'david_downs@harvard.edu',
+                'david_bonner@harvard.edu',
+                'douglashall@g.harvard.edu'
+            },
+            add_members={'david_bonner@harvard.edu'},
+            delete_members=set(),
+            members_count=3
+        )
+
+    @override_settings(IGNORE_WHITELIST=False)
+    def test_sync_listserv_membership_ignore_whitelist_false(self):
+        self._call_sync_listserv_membership(
```

Comment:
This looks like the same test as `test_sync_listserv_membership_whitelist_not_set` and `test_sync_listserv_membership_unsubscribed`...?